### PR TITLE
chore(CIS): Ensure logging is configured

### LINKED ISSUE #1011: CIS 4.2.1.2 Ensure logging is configured
4.2.1.2 Ensure logging is configured (Not Scored)

```
cookbook_file 'Set /etc/rsyslog.d/60-CIS.conf' do
  path '/etc/rsyslog.d/60-CIS.conf'
  source 'etc/rsyslog.d/60-CIS.conf'
end
```

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
**Reason for Change**:

CIS 4.2.1.2 Ensure logging is configured

**Issue Fixed**:

Fixes #1011 

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: test/e2e/kubernetes/kubernetes_test.go [hunk 1]
```diff
@@ -201,7 +201,7 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 					Expect(err).NotTo(HaveOccurred())
 				}
 			} else {
-				Skip("root password validation only works on ubuntu distro until this lands in a VHD")
+				Skip("net config validation only works on ubuntu distro until this lands in a VHD")
```
Comment (CecileRobertMichon) [inline test/e2e/kubernetes/kubernetes_test.go]: just fixing a copy paste typo from a previous PR

File: parts/k8s/cloud-init/artifacts/cis.sh [hunk 2]
```diff
@@ -45,6 +45,7 @@ assignFilePermissions() {
     chmod 600 /etc/shadow- || exit $ERR_CIS_ASSIGN_FILE_PERMISSION
     chmod 600 /etc/group- || exit $ERR_CIS_ASSIGN_FILE_PERMISSION
     chmod 644 /etc/sysctl.d/60-CIS.conf || exit $ERR_CIS_ASSIGN_FILE_PERMISSION
+    chmod 644 /etc/rsyslog.d/60-CIS.conf || exit $ERR_CIS_ASSIGN_FILE_PERMISSION
```
Comment (jackfrancis) [inline parts/k8s/cloud-init/artifacts/cis.sh]: Why are we enforcing file mode requirements here + cloud-init?

Comment (jackfrancis) [reply parts/k8s/cloud-init/artifacts/cis.sh]: Nevermind, VHD CI

**Comment by jackfrancis** [comment]
lgtm

File: packer/install-dependencies.sh [hunk 3]
```diff
@@ -17,7 +19,7 @@ echo "Starting build on " `date` > ${RELEASE_NOTES_FILEPATH}
 echo "Using kernel:" >> ${RELEASE_NOTES_FILEPATH}
 cat /proc/version | tee -a ${RELEASE_NOTES_FILEPATH}
 cp $SYSCTL_CONFIG_SRC $SYSCTL_CONFIG_DEST
-sysctl_reload 20 5 10
```
Comment (mboersma) [inline packer/install-dependencies.sh]: Is this a related change?

Comment (mboersma) [reply packer/install-dependencies.sh]: Never mind, I found the reference to #1061 in another CIS PR.
